nixos/modules/services/web-apps/coder.nix

```diff
@@ -159,7 +176,7 @@ in {
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
 
-      environment = {
+      environment = config.environment.extra // {
```

Comment:
Hello,

Hello, 

I am not sure it is an issue, but I have an error with the module.

I think it should be `cfg.environment.extra` instead of `config.environment.extra`.

Regards.

Reply:
Can you open a PR fixing it?

Reply:
Thank you for the quick reply. 

I am not 100% confident with both NIX and the PR process. 
But, I will manage to do it during the week-end.

Reply:
Could not find to do it myself, but @kylecarbs did it : #305993.